adventure.py

```diff
@@ -26,10 +28,24 @@
 else:
     with_colors = False
 
-if platform.system() != "Windows":
+if platform.system() != "Windows": # compatibility fixes
     clear_command = "clear"
 else:
     clear_command = "cls"
+
+with_custom = input("Generate map or use custom?") # asking user to either generate or use custom map
+if with_custom.lower() in "generate":
+    dim = int(input("Tile dimension?\n"))  # getting world dimensions from user
+    world = world_tile(dim, "world", with_colors, False,"") # creating "world" object in "table" class with user input
+    system(clear_command) # clearing screen to prepare for game
+else:
+    dim = 5
```

Comment:
Why 5?

Reply:
Arbitrary. Will be overwritten when the custom map file is opened and read, but needed an integer placeholder or the code screams.